### PR TITLE
Bump up version to 2.0.1

### PR DESCRIPTION
**Pull request by slightair**
for release 2.0.1

### CODE REVIEW

File: Example/Tests/PURLogStoreTest.swift [hunk 1]
```diff
@@ -148,7 +148,7 @@ class PURLogStoreTest: XCTestCase {
         for i in 1...1000 {
             addTestLog(PURLog(tag: "testC.peach", date: Date(), userInfo: [:]), output: outputC, description: "\(i)")
         }
-        waitForExpectations(timeout: 10.0, handler: nil)
+        waitForExpectations(timeout: 15.0, handler: nil)
```
Comment (giginet) [inline Example/Tests/PURLogStoreTest.swift:151]: Why do you increase duration?

Comment (slightair) [reply Example/Tests/PURLogStoreTest.swift:151]: Removed databaseReadWriteCompletionQueue In this PR https://github.com/cookpad/puree-ios/pull/17.
So the completion block run on the main-thread and extend time of writing all logs.

https://travis-ci.org/cookpad/puree-ios/builds/172282111

```
✓ testStressTest (9.247 seconds)
```

Maybe TravisCI environment slower than local machines (My iMac average 5sec).
I expect test time about 10sec, so change this timeout.

Comment (giginet) [reply Example/Tests/PURLogStoreTest.swift:151]: I got, no problem

**Comment by giginet** [comment]
LGTM

**Comment by slightair** [comment]
Thanks!

**Pull request merged**